docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -140,6 +141,17 @@ TIP: Platinum and Enterprise customers can customize these notifications using t
 [role="screenshot"]
 image::images/install-endpoint/behavior-protection.png[Detail of behavior protection section.]
 
+[discrete]
+[[attack-surface-reduction]]
+== Attack surface reduction
+
+This section helps you reduce vulnerabilities that attackers can target in Windows endpoints.
```

Comment:
```suggestion
This section helps you reduce vulnerabilities that attackers can target on Windows endpoints.
```

Reply:
I get that "in" is the more grammatically correct preposition to use since the vulnerabilities are figuratively/virtually/maybe even physically within the endpoint, instead of top or around it. But "on" sounds more colloquially correct. WDYT?

Reply:
Agree, I think something like a vulnerability would exist "in" Windows (like _in_ its code) but "on" an endpoint.